packages/scaffold-ui/src/partials/w3m-header/index.ts

```diff
@@ -167,6 +168,21 @@ export class W3mHeader extends LitElement {
     }
   }
 
+  private rightHeaderTemplate() {
+    if (this.view !== 'Account') {
+      this.closeButtonTemplate()
+    }
+
+    return html`<wui-flex>
+      <wui-icon-link
+        icon="clock"
+        @click=${() => RouterController.push('SmartSessionList')}
+        data-testid="w3m-header-smart-sessions"
+      ></wui-icon-link>
+      ${this.closeButtonTemplate()}
+    </wui-flex> `
```

Comment:
This is wrong, `SmartSessionList` will not be available unless the dapp imports the package. 
I should hide this with a feature flag.

Reply:
Done